Safety/Src/PWM.cpp

```diff
@@ -3,31 +3,34 @@
 #include "PWM.hpp"
 #define MAX_CHANNELS 10
 #define NUM_AVAILABLE_CHANNELS 8
+#define DSHOT_DATA_FRAME_LEN 16
+#define DSHOT_DMA_BUFFER_SIZE 18
+#define DSHOT_MAX_THROTTLE 2047
+#define DSHOT_BIT_1 240
+#define DSHOT_BIT_0 120
```

Comment:
This isn't necessarily true, as it depends on the dshotType, right? or maybe that calculation is supposed to be done elsewhere? I would also appreciate the name being a little more descriptive of what it actually represents.